deploy

```diff
@@ -1,223 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why isn't this considered a move?

Reply:
IDK probably git don't recognize changes in this case